pkg/tfvars/tfvars.go

```diff
@@ -64,16 +63,9 @@ func TFVars(cfg *types.InstallConfig, bootstrapIgn, masterIgn string) ([]byte, e
 				}
 			}
 		case "worker":
-			config.Workers += replicas
 			if m.Platform.AWS != nil {
 				config.AWS.Worker = aws.Worker{
-					EC2Type:     m.Platform.AWS.InstanceType,
 					IAMRoleName: m.Platform.AWS.IAMRoleName,
-					WorkerRootVolume: aws.WorkerRootVolume{
```

Comment:
Is this why workers are now 16gb volumes?

Reply:
> Is this why workers are now 16gb volumes?

This is just passing data to Terraform, and worker creation is via the cluster-API now.  So I don't see a connection.